src/bin/WinKeyServer/main.cpp

```diff
@@ -82,6 +92,14 @@ __declspec(dllexport) LRESULT CALLBACK KeyboardEvent(int nCode, WPARAM wParam, L
             }
             return 1;
         }
+        else

```

Comment:
Additionally moving this in the else block makes sure that we don't send a fake key press if the `alt` or `meta` down itself was already caught.